### PR TITLE
nsd 4.1.6

### PR DESCRIPTION
**Pull request by Habbie**

### CODE REVIEW

File: Library/Formula/nsd.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Nsd < Formula
   desc "Name server daemon"
   homepage "https://www.nlnetlabs.nl/projects/nsd/"
-  url "https://www.nlnetlabs.nl/downloads/nsd/nsd-4.1.3.tar.gz"
-  sha256 "097cb295cdd3e8a73a8afac343caf0fca11b72b2214b022689ddc423312d17e5"
+  url "https://www.nlnetlabs.nl/downloads/nsd/nsd-4.1.5.tar.gz"
+  sha256 "105dda21fab68b150f1f4d045211869b13becadc52be830b9c0093f8e9f01baa"
   revision 1
```
Comment (bfontaine) [inline Library/Formula/nsd.rb]: Please remove this line as well; thanks!

Comment (Habbie) [inline Library/Formula/nsd.rb]: ah yes, done!

**Comment by Habbie** [comment]
4.1.6 was just released, please hold

**Comment by Habbie** [comment]
updated to 4.1.6

**Comment by DomT4** [comment]
And :boat:'ed in https://github.com/Homebrew/homebrew/commit/320c9abb77605c106d0ed6595dff05efb0231432. Thank you!

**Pull request closed without merging**